applications/channel/channel-plugins/org.csstudio.pvmanager.formula.channelfinder/src/org/csstudio/pvmanager/formula/channelfinder/CFQueryFunction.java

```diff
@@ -60,18 +61,18 @@ public Object calculate(List<Object> args) {
         if (currentQuery == null || !((VString) args.get(0)).getValue().equals(currentQuery.getValue())) {
             currentQuery = (VString) args.get(0);
             serviceMethod = ServiceRegistry.getDefault().findServiceMethod("cf/find");
-            serviceMethod.executeMethod(Collections.<String, Object>singletonMap("query", currentQuery), new WriteFunction<Map<String, Object>>(){
+            serviceMethod.executeAsync(Collections.<String, Object>singletonMap("query", currentQuery), new Consumer<Map<String, Object>>(){
```

Comment:
@shroffk I made a few changes in the channel-plugins directory you might like to check